requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
+pandas
```

Comment:
`pytest` is missing

```suggestion
pandas
pytest

```

Reply:
actually yes, tests are working with pytest right now closely. 
Also, I will add an .md file to the test folder that briefly guides a user through the test files.